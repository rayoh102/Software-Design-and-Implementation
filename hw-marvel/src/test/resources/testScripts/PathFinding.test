# Tests if we can find a single hop path

# Create a Graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add another node
AddNode graph1 n2

# Add edge
AddEdge graph1 n1 n2 e1

# Find path
FindPath graph1 n1 n2