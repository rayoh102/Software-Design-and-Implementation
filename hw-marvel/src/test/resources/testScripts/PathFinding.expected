# Tests if we can find a single hop path

# Create a Graph
created graph graph1

# Add a node
added node n1 to graph1

# Add another node
added node n2 to graph1

# Add edge
added edge e1 from n1 to n2 in graph1

# Find path
path from n1 to n2:
n1 to n2 via e1